mcv/hostname.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+
+def _read_stripped_lines(f):
+    return [l.rstrip() for l in f.readlines()]
```

Comment:
Not recommend to put a mutative stateful operation in here; prefer stateless operations, e.g. either

1. Open, read and close the file handle all within this function, return a pure list of stripped lines
2. Open, read and close the file handle elsewhere.  Then make this a pure function `:: [Line] -> [Line]`

```python
def stripped_lines(path):
    with open(path) as f:
        return [l.rstrip() for l in